pages/linux/chrt.md

```diff
@@ -0,0 +1,20 @@
+# chrt
+
+> Manipulate the real-time attributes of a process.
+> More information: <https://man7.org/linux/man-pages/man1/chrt.1.html>.
+
+- Display attributes of a process:
+
+`chrt --pid {{PID}}`
+
+- Display attributes all threads of a process:
```

Comment:
Should this be `Display attributes of all threads of a process`?

Reply:
Haha, yes, for sure.